edgedb/errors/_base.py

```diff
@@ -198,3 +324,9 @@ def _severity_name(severity):
 EDGE_SEVERITY_ERROR = 120
 EDGE_SEVERITY_FATAL = 200
 EDGE_SEVERITY_PANIC = 255
+
+
+LINESEP = os.linesep
+ENV_ON_FLAGS = {"1", "yes", "y", "true", "t", "on"}
```

Comment:
Should we maybe copy edgedb server flag policy and have an enum here? `auto | enabled | disabled` should work